PCGen-base/code/src/java/pcgen/base/lang/StringUtil.java

```diff
@@ -63,7 +65,13 @@ private StringUtil()
 	 */
 	public static String join(Collection<?> collection, String separator)
 	{
-		return joinToStringBuilder(collection, separator).toString();
+		if (collection == null)
```

Comment:
maybe use something like nullToEmpty here? If we don't want to include a larger library write our own (or copy the existing one)